CHANGELOG.md

```diff
@@ -20,6 +20,9 @@
 * None.
 
 #### Enhancements
+* Update README configuration for new projects. The `included` used to have the old `Source` path, 
+  the new default is `Sources`.  
+  [chibombo](https://github.com/chibombo)
```

Comment:
No need for a changelog entry. Please remove it.

Reply:
Done 🚀